dev/com.ibm.ws.security.jwt_fat.builder/fat/src/com/ibm/ws/security/jwt/fat/builder/utils/BuilderHelpers.java

```diff
@@ -81,7 +81,7 @@ public static JSONObject setDefaultClaimsWithEncryption(String builderId, String
         settings.remove(HeaderConstants.ALGORITHM);
         settings.put(HeaderConstants.ALGORITHM, KeyMgmtKeyAlg);
         settings.put(HeaderConstants.ENCRYPTION, contentEncryptAlg);
-        settings.put(HeaderConstants.TYPE, "JWT");
+        settings.put(HeaderConstants.TYPE, "JOSE");
         settings.put(HeaderConstants.CONTENT_TYPE, "jwt");
```

Comment:
No changes needed now, but may need to be set to "JWT" if causing problem with legacy implementations per https://tools.ietf.org/html/rfc7519#section-5.1.